spec/features/consumer/shopping/shopping_spec.rb

```diff
@@ -50,6 +50,14 @@
         page.should have_selector "option[selected]", text: 'turtles'
       end
 
+      it "displays additional notes for order cycle" do
+        exchange1.update_attribute :pickup_time, "hedgehog"
+        exchange1.update_attribute :pickup_instructions, "Tickle my belly before you pick me up"
+        visit shop_path
+        select "hedgehog", :from => "order_cycle_id"
+        page.should have_content exchange1.pickup_instructions
```

Comment:
I noticed that your specs are often in a separate commit after the feature. Have you encountered TDD (test-driven development) before? I've found it a useful practice, helping me to write testable code in the first place and helping to guide software design, too. It also helps solve the problem of writing a spec that passes, but doesn't actually test the feature. With TDD you start with a failing spec for the unimplemented feature, and then when the spec goes green you can have confidence that the spec is actually testing the feature. I'm a fan, so just putting it on your radar in case it's something you'd like to explore.